features/csharp/csharp_persistence.feature

```diff
@@ -75,22 +62,11 @@ Feature: Unity Persistence
     And the exception "message" equals "PersistDeviceId"
     And the error payload field "events.0.device.id" equals the stored value "device_id"
 
-
   Scenario: Max Persisted Events
-    When I clear the Bugsnag cache
-    And I wait for 3 seconds
+    When I run the game in the "MaxPersistEvents" state
+    And I wait for 45 seconds
```

Comment:
It's already set to 3, which i think is a good number to avoid false positives. iOS just seems to have much longer timeouts for requests than other platforms. Perhaps i should add a custom step that only waits that long on iOS?

Reply:
We decided to rework the way this test is written so that the fixture checks for the correct number of persisted events in the file system